### PR TITLE
community/events: Adding Ceph Days Asia 2024 event page

### PR DESCRIPTION
**Pull request by ssdohammer-sl**
Add an event for Ceph Days Asia 2024.

Signed-off-by: Sungmin Lee sung_min.lee@samsung.com

### CODE REVIEW

File: src/en/community/events/2024/ceph-days-asia/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+<a class="button" href="https://docs.google.com/forms/d/1_jrjU_PzbnqT9M7yO2GKq1RLYmh6tJmsnjKwhmfpOrc/edit">CFP is now Open!</a>
+<a class="button" href="#">Registeration Coming Soon</a>
+
+## Important Dates
+
+- CFP Opens: 2024-05-27
+- CFP Closes: 2024-07-05
+- Speakers receive confirmation of acceptance: 2024-07-12
+- Schedule Announcement: 2024-07-26
```
Comment (myoungwon) [inline src/en/community/events/2024/ceph-days-asia/index.md:33]: 7/24 ?

**Comment by myoungwon** [comment]
@dvanders @neha-ojha @jdurgin Please take a look.

File: src/en/community/events/2024/ceph-days-asia/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: Ceph Days Asia 2024
+date: 2024-09-03
+end: 2024-09-04
+location: 140 Gwanggyojungang-ro, Yeongtong-gu, Suwon-si, Gyeonggi-do, Republic of Korea
+venue: Suwon Convention Center
+image: "/assets/bitmaps/ceph-days.png"
+sponsors:
+
```
Comment (myoungwon) [inline src/en/community/events/2024/ceph-days-asia/index.md:9]: Please add "hosted by samsung"

Comment (ssdohammer-sl) [reply src/en/community/events/2024/ceph-days-asia/index.md:13]: A host info and its logo is added.

**Review by anthonyeleven — APPROVED** [review]

**Pull request merged**